include/fastdds/dds/xtypes/type_representation/detail/dds_xtypes_typeobjectCdrAux.hpp

```diff
@@ -24,13 +24,13 @@
 
 #include "dds_xtypes_typeobject.hpp"
 
-constexpr uint32_t eprosima_fastdds_dds_xtypes_CompleteStructMember_max_cdr_typesize {1464UL};
+constexpr uint32_t eprosima_fastdds_dds_xtypes_CompleteStructMember_max_cdr_typesize {1436UL};
```

Comment:
Do you know what is provoking this change?

Reply:
This can also be caused for allocating less samples in unbounded collections